Documentation/MAX32655.html

```diff
@@ -55,27 +55,41 @@ <h2>Example Applications</h2>
 			<li><a href='../Examples/MAX32655/BLE_otac/README.md'>BLE_otac</a> - Bluetooth Data Client with Over-the-Air Updates</li>
 			<li><a href='../Examples/MAX32655/BLE_otas/README.md'>BLE_otas</a> - Bluetooth Data Server with Over-the-Air Updates</li>
-			<li><a href='../Examples/MAX32655/BLE_ctr/README.md'>BLE4_ctr</a> - Bluetooth v4.2 Controller</li>
+			<li><a href='../Examples/MAX32655/BLE_periph/README.md'>BLE_periph</a> - Bluetooth Peripheral</li>
+			<li><a href='../Examples/MAX32655/BLE4_ctr/README.md'>BLE4_ctr</a> - Bluetooth v4.2 Controller</li>
+			<li><a href='../Examples/MAX32655/BLE5_ctr/README.md'>BLE5_ctr</a> - Bluetooth v5.2 Controller</li>
 			<li><a href='../Examples/MAX32655/Bootloader/README.md'>Bootloader</a></li>
 			<li><a href='../Examples/MAX32655/CRC/README.md'>CRC</a> - Cyclic Redundancy Check</li>
 			<li><a href='../Examples/MAX32655/DMA/README.md'>DMA</a> - Direct Memory Access</li>
+			<li><a href='../Examples/MAX32655/Dual_core_sync/Dual_core_sync_arm/README.md'>Dual_core_sync</a> - Dual Core Inter-Communication</li>
 			<li><a href='../Examples/MAX32655/Flash/README.md'>Flash</a> - Flash Memory</li>
+			<li><a href='../Examples/MAX32655/Flash_CLI/README.md'>Flash-CLI</a> - Flash Memory - Demonstration of storing non-volatile data</li>
 			<li><a href='../Examples/MAX32655/FreeRTOSDemo/README.md'>FreeRTOSDemo</a></li>
-			<li><a href='../Examples/MAX32655/GPIO/README.md'>GPIO</a> - General Purpose Input/Output</li>
 			<li><a href='../Examples/MAX32655/FTHR_I2C/README.md'>FTHR_I2C</a> - I2C for Featherboard</li>
+			<li><a href='../Examples/MAX32655/GPIO/README.md'>GPIO</a> - General Purpose Input/Output</li>
 			<li><a href='../Examples/MAX32655/Hello_World/README.md'>Hello_World</a></li>
 			<li><a href='../Examples/MAX32655/Hello_World-riscv/README.md'>Hello_World-riscv</a></li>
 			<li><a href='../Examples/MAX32655/I2C/README.md'>I2C</a> - The Standard 2-Wire Serial Protocol</li>
+			<li><a href='../Examples/MAX32655/I2C_MNGR/README.md'>I2C_MNGR</a> - Demonstration of managing simultaneous I2C transaction requests</li>
+			<li><a href='../Examples/MAX32655/I2C_SCAN/README.md'>I2C_SCAN</a> - Demonstration of discovering I2C devices on the bus</li>
+			<li><a href='../Examples/MAX32655/I2C_Sensor/README.md'>I2C_Sensor</a> - Demonstration of communicating with an I2C temperature sensor</li>
 			<li><a href='../Examples/MAX32655/I2S/README.md'>I2S</a> - Inter-IC Sound</li>
+			<li><a href='../Examples/MAX32655/I2S_Playback/README.md'>I2S_Playback</a> - Demonstration of playback of audio sample using I2S</li>
 			<li><a href='../Examples/MAX32655/ICC/README.md'>ICC</a> - Internal Cache Controller</li>
 			<li><a href='../Examples/MAX32655/LP/README.md'>LP</a> - Low Power</li>
+			<li><a href='../Examples/MAX32655/LPCMP/README.md'>LPCMP</a> - Low Power Comparator</li>
 			<li><a href='../Examples/MAX32655/Pulse_Train/README.md'>Pulse_Train</a></li>
```

Comment:
I believe RF_Test is missing here.

Reply:
This was intentional.  There is a BLE_fcc that seems to being the same thing.  I need Kevin to weigh in on if both or needed.